docs/cloud-native-security/d4c-policy-guide.asciidoc

```diff
@@ -0,0 +1,94 @@
+- *Drift Detection & Prevention:* The second default policy is designed to create alerts when container drift is detected. It is called `executableChanges`, and selects all `createExecutable` and `modifyExecutable` events. It is mapped to a response called `executableChanges`, which specifies that the selected events should create alerts, which will be sent to your {es} cluster. This policy can also be modified to block drift operations by changing the response action to block.
+
+image::images/d4c-policy-editor.png[The defend for containers policy editor with the default policies]
+
+
+[[d4c-selectors-glossary]]
+[discrete]
+== Selectors
+A selector requires a name and at least one operation. It will select all events of the specified operation types, unless you also include _conditions_ to narrow down the selection. Some conditions are available for both `file` and `process` selectors, while some are only available for one type of selector.
+
+[discrete]
+=== Common conditions
+These conditions are available for both `file` and `process` selectors.
+
+[cols="1,1", options="header"]
+|===
+| Name | Description
+| containerImageFullName | A list of container full image names to match on, for example: `docker.io/nginx`.
+| containerImageName | A list of container image names to match on. For example: `nginx`.
+| containerImageTag | A list of container image tags to match on. For example: `latest`.
+| kubernetesClusterId | A list of Kubernetes cluster IDs to match on. For consistency with KSPM, the `kube-system` namespace's UID is used as a cluster ID.
+| kubernetesClusterName | A list of Kubernetes cluster names to match on.
+| kubernetesNamespace | A list of Kubernetes namespaces to match on.
+| kubernetesPodName | A list of Kubernetes pod names to match on. Trailing wildcards supported.
+| kubernetesPodLabel | A list of resource labels. Trailing wildcards supported (value only), for example: `key1:val*`.
+|===
+
+[discrete]
+=== File-selector conditions
+These conditions are available only for `file` selectors.
+
+[cols="1,1", options="header"]
+|===
+| Name | Description
+| operation | The list of system operations to match on. Options include `createExecutable`, `modifyExecutable`, `createFile`, `modifyFile`, `deleteFile`.
+| ignoreVolumeMounts | If set, ignores file operations on ALL volume mounts.
+| ignoreVolumeFiles | If set, ignores operations on file mounts only. For example: mounted files, `configMaps`, and secrets.
+| targetFilePath | A list of file paths to include. Paths are absolute and wildcards are supported. The * wildcard matches any sequence of characters within a single directory, while the ** wildcard matches any sequence of characters across multiple directories and subdirectories. Note: In order to ensure precise targeting of file integrity monitoring operations, a `TargetFilePath` is required whenever the `deleteFile`, `modifyFile`, or `createFile` operations are used within a selector.
```

Comment:
The Note at the end should be a separate paragraph and formatted per Asciidoc, not just added to the end of another paragraph. Looks like maybe you were in the middle of doing that, since there's an empty `NOTE:` line below?